extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcProviderClientImpl.java

```diff
@@ -42,6 +42,14 @@
 import io.vertx.mutiny.ext.web.client.WebClient;
 
 public class OidcProviderClientImpl implements OidcProviderClient, Closeable {
+
+    private enum TokenOperation {
+        Get,
```

Comment:
Better make them all uppercase, since they are constants

Reply:
OK

Reply:
I am fine with it, but I can't see it as an improvement @gastaldi , it is private enum used for a log message, so the capitalization helped the log message to be more natural.

Reply:
Hi @gastaldi @michalvavrik I just pushed the update, Michal, I did not see your last message, yeah, it is ok either way I guess with the internal code